data/explorations/audio_test/audio_test.yaml

```diff
@@ -17,11 +17,11 @@ states:
       audio_translations:
         en:
           filename: 'test_audio_2_en.mp3'
-          file_size_bytes: 2
+          file_size_bytes: 100
```

Comment:
Yeah sorry I kept thinking in kB when we were using bytes so these values were way too low, using actual values now